### ISSUE TITLE
Hosting default options for NEWCOMERS.

### ISSUE THREAD

**Issue by erik90mx** [drawpile/Drawpile#712]
Hi,
This is a situation that I see a lot when newcomer try to host a session to share the canvas:

- They host a session in their own computer.
- They don’t have the necessary router configuration (even if the Drawpile web told about it, not every user has the knowledge to change the configuration, or not everyone reads it before use Drawpile to draw)
- They stay in a room thinking that “nobody wants to enter”, when the reality is that none actually can, and they can’t be advised.
- After trying to host in the same way several times, they just leave without trying the other rooms available.

This is a very common situation, that leads in newcomers trying Drawpile and leaving it thinking “it didn’t worked for me”.

**EDIT**

Some minutes has passed and I found this situation already:
![image](https://user-images.githubusercontent.com/3485891/54869696-02c35400-4d62-11e9-98a1-fee4fd561852.png)

To avoid that, it will be a very good option:

- Have some warnings or advices in the Host window (for the Listing and Hosting section)
- Have default hosting preferences selected to host in drawpile.net
- Hide all the options in a “customize/advanced” button way, to just provide the Session necessary options straightforward.

(example of default preferences "plug and play")
![image](https://user-images.githubusercontent.com/3485891/54869275-b117cb00-4d5b-11e9-8106-e05be801312d.png)

Regards

**Comment by callaa** [comment]
Yes, there seems to be a lot of confusion about how hosting works, and Drawpile should offer more help. There's a hosting guide on the website but it seems people don't read it.

So, a couple of ideas I've been thinking about:

 * Add a "how do I choose?" link to the Hosting box that takes to the hosting help page
 * Add a new intro section to the help page that explains the best option to use for each use case

Another one that's been on my TODO list for a long time is an improvement to the listing server. When you announce a listing, it could try to connect to your server. If it can't connect, it responds with an error message explaining the situation.

**Comment by erik90mx** [comment]
The TODO feature sounds very handy, but it will involves that after failed the user knows what happens (and this error message can advice with the help link that you mentioned too), if we try to focus that the user makes the hosting right since the begining before failing, a "test conection" feature in the hosting windows sounds handy beforehand.

Regards

**Comment by Wade821** [comment]
Partially implemented on list server:
https://github.com/drawpile/listserver/commit/519ce913ba7b9cc9ffcdecdd657c451c69373fe4

**Comment by askmeaboutlo0m** [comment]
The host dialog has been rejigged to a large degree, providing more sensible defaults and information. Hosting on your own computer warns that additional setup is required and links to instructions.
